package.json

```diff
@@ -21,8 +21,8 @@
     "babel-preset-es2015": "^6.24.1",
     "eslint": "^3.15.0",
     "jest": "^20.0.4",
-    "pre-commit": "^1.1.3",
-    "webpack": "^2.4.1"
+    "pre-commit": "^1.2.2",
+    "webpack": "^2.6.1"
```

Comment:
These dev dependency changes aren't strictly necessary.  Not a huge deal, but please try to avoid updating unless necessary.